cli/login/login.go

```diff
@@ -384,3 +384,40 @@ func isSupportedCommand(command string) bool {
 	}
 	return false
 }
+
+// WithApiKeyInteractive attempts to read an API key from the buildbuddy config
```

Comment:
I think this func could be simplified a bit, which would hopefully make it a little more generally useful:
- Return an error instead of warning - then the caller could log whatever suggestion is appropriate (instead of accepting a bool param that controls the logging behavior)
- Return the API key as a string instead of setting it on the outgoing context (and rename to `GetAPIKeyInteractive`)

Reply:
Good point, fixed (it now also reads the API key from env)